lapack/testlapack/dlansy.go

```diff
@@ -38,17 +38,25 @@ func DlansyTest(t *testing.T, impl Dlansyer) {
 					if lda == 0 {
 						lda = n
 					}
+					// Allocate n×n matrix A and fill it with random numbers.
```

Comment:
`// Allocate n×n matrix A and fill it.` since they are not always random (unless you take [this definition](https://www.xkcd.com/221/) of random).

Reply:
Done.